src/js/letters/containers/AddressSection.jsx

```diff
@@ -155,6 +155,8 @@ export class AddressSection extends React.Component {
   handleCancel = () => {
     this.setState({
       isEditingAddress: false,
+      errorMessages: {},
```

Comment:
@cvalarida this will make you unhappy but I didn't end up defining an initial state object because we don't want to reset `hadLoadedAddress` and I figured that defining an initial state just for `shouldValidate` and `errorMessages` was adding complexity.

Reply:
Ah, right. I didn't think about that. Good call!

Reply:
er, `hasLoadedAddress`